### PR TITLE
replace constant and delete fixtures

### PR DESCRIPTION
**Pull request by annashamray**
**Changes;**
1. delete all fixtures
1. in INSTALL.rst remove command to load from fixtures
1. in `brondatum_archiefprocedure_objecttype` field of `ResultaatType` model use choice `ZaakobjectTypes` from zds_schema

### CODE REVIEW

**Comment by sergei-maertens** [comment]
add to whitelist

File: src/ztc/datamodel/models/resultaattype.py [hunk 1]
```diff
@@ -6,8 +6,9 @@
 import requests
 from zds_schema.constants import (
     Archiefnominatie,
-    BrondatumArchiefprocedureAfleidingswijze as Afleidingswijze, ObjectTypes
+    ZaakobjectTypes as Afleidingswijze, ObjectTypes
```
Comment (sergei-maertens) [inline src/ztc/datamodel/models/resultaattype.py]: that's a broken import - the alias should point to `BrondatumArchiefprocedureAfleidingswijze`, and `ObjectTypes` should be replaced with `ZaakobjectTypes`

Comment (annashamray) [reply src/ztc/datamodel/models/resultaattype.py]: fixed it

File: src/ztc/datamodel/models/resultaattype.py [hunk 2]
```diff
@@ -6,8 +6,10 @@
 import requests
 from zds_schema.constants import (
     Archiefnominatie,
-    BrondatumArchiefprocedureAfleidingswijze as Afleidingswijze, ObjectTypes
+    BrondatumArchiefprocedureAfleidingswijze as Afleidingswijze,
+    ZaakobjectTypes
 )
+
```
Comment (sergei-maertens) [inline src/ztc/datamodel/models/resultaattype.py]: this blank line shouldn't be here, according to `isort`

**Comment by sergei-maertens** [comment]
test this please

**Comment by sergei-maertens** [comment]
test this please

File: src/ztc/datamodel/models/resultaattype.py (hunk 2 above)
Comment (annashamray) [reply src/ztc/datamodel/models/resultaattype.py]: fixed it

**Pull request merged**